dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -57,72 +58,124 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
     void AppNotificationManager::Register()
     {
-        winrt::guid storedComActivatorGuid{ GUID_NULL };
-        if (!PushNotificationHelpers::IsPackagedAppScenario())
+        HRESULT hr{ S_OK };
+        std::wstring appId{};
```

Comment:
See appId being used a lot everywhere. Maybe just introduce a member variable m_appId and initialize that in the ctor? Then we can delete all this redundant code and reuse m_appId everywhere.

Reply:
good call, done